core/src/main/resources/reference.conf

```diff
@@ -33,6 +33,10 @@ akka.persistence.cassandra {
 
   session-name = ""
 
+  # The config path of the datastax-java-driver configuration.
+  # see: https://docs.datastax.com/en/developer/java-driver/4.3/manual/core/configuration/#quick-overview
+  datastax-java-driver-config = "datastax-java-driver"
```

Comment:
I tried this with a custom datastax-java-driver2 section, and that works. Nice because then it's possible with different driver configs for different plugin instances.

Was also thinking about if this should be an inline section, always defining it inside the plugin config. Maybe best to not deviate from the "standard" way of defining it in `datastax-java-driver` section.

Was also thinking about the config reload feature that is provided by the driver. I guess that is if the application.conf is defined in a file. Not sure we need to support that? We could say that if this is defined as `""` we just use plain default loader.

Reply:
did we need this because we need to override things not in the profile?

Reply:
I think motivation is explained in issue #680
Let me know if still unclear.